### PR TITLE
linux support for fontbm

### PR DESCRIPTION
**Pull request by mkellner**
Add fontbm setup #14 support for Linux.

### CODE REVIEW

File: src/toolbox/setup/fontbm.ts [hunk 1]
```diff
@@ -20,15 +21,35 @@ export default async function (): Promise<void> {
...
-    spinner.succeed()
+    if (OS == 'darwin') {
+      spinner.start('Cmake required, installing with Homebrew')
+   	  await system.exec('brew install cmake')
+      spinner.succeed()
+	}
+	else if (OS == 'linux') {
+      spinner.start('CMake required, installing with apt')
+      await execWithSudo(
+        'apt-get install --yes cmake',
```
Comment (HipsterBrown) [inline src/toolbox/setup/fontbm.ts:32]: When testing this out in a fresh Linux VM, I had trouble during the `cmake .` step due to missing compiler flags. The fix was to ensure `build-essential` was installed before running `cmake`. While this is probably in most Linux devs' environments already, we can help those few that don't have it yet. 

```suggestion
        'apt-get install --yes build-essential cmake',
```

**Review by HipsterBrown — CHANGES_REQUESTED** [review]
Thanks for adding linux support @mkellner! :clap:

Just a few suggestions for consistency and smoothing out any potential cmake issues. Then it looks good to ship! 🚀

**Comment by mkellner** [comment]
Thank you for the suggestions. I've incorporated them.

**Review by HipsterBrown — APPROVED** [review]

**Comment by HipsterBrown** [comment]
@mkellner This has been released in v0.6.0! Thanks again for contributing 👏 🚀

**Pull request merged**